Apps/W1/Shopify/app/src/Logs/Tables/ShpfySkippedRecord.Table.al

```diff
@@ -0,0 +1,135 @@
+namespace Microsoft.Integration.Shopify;
+
+using System.Reflection;
+using Microsoft.Utilities;
+
+/// <summary>
+/// Table Shpfy Skipped Record (ID 30159).
+/// </summary>
+table 30159 "Shpfy Skipped Record"
+{
+    Caption = 'Shpfy Skipped Record';
+    DataClassification = SystemMetadata;
+    Access = Internal;
+
+    fields
+    {
+        field(1; "Entry No."; Integer)
+        {
+            AutoIncrement = true;
+            Caption = 'Entry No.';
+            DataClassification = SystemMetadata;
+        }
+        field(2; "Shopify Id"; BigInteger)
+        {
+            Caption = 'Skipped Record Id';
+            DataClassification = SystemMetadata;
+        }
+        field(3; "Table ID"; Integer)
+        {
+            Caption = 'Table ID';
+            DataClassification = SystemMetadata;
+
+            trigger OnValidate()
+            begin
+                "Table Name" := GetTableCaption();
+            end;
+        }
+        field(4; "Table Name"; Text[250])
+        {
+            Caption = 'Table Name';
+            DataClassification = SystemMetadata;
+        }
+        field(5; "Record ID"; RecordID)
+        {
+            Caption = 'Record ID';
+            DataClassification = SystemMetadata;
+
+            trigger OnValidate()
+            begin
+                Description := GetRecDescription();
+            end;
+        }
+        field(6; Description; Text[250])
+        {
+            Caption = 'Description';
+            DataClassification = SystemMetadata;
+        }
+        field(7; "Skipped Reason"; Text[250])
+        {
+            Caption = 'Skipped Reason';
+            DataClassification = SystemMetadata;
+        }
+        field(8; "Created On"; DateTime)
```

Comment:
Why are these two Created fields needed? We have system created by, and at fields?

Reply:
makes sense its not needed